config/jobs/cert-manager/cert-manager/cert-manager-periodics.yaml

```diff
@@ -51,7 +56,9 @@ periodics:
       preset-enable-all-feature-gates-disable-ssa: "true"
       preset-ginkgo-skip-default: "true"
       preset-make-volumes: "true"
+      preset-retry-flakey-tests: "true"
```

Comment:
note: Adding retry-flakey-tests to every E2E test is recurring. It seems like if we want it for _some_ E2Es, we probably want it on them all?